airbyte-integrations/connectors/source-jira/source_jira/schemas/application_roles.json

```diff
@@ -1,57 +1,48 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "type": "object",
+  "$schema": "http://json-schema.org/schema#",
   "properties": {
-    "key": {
-      "type": "string",
-      "description": "The key of the application role."
+    "defaultGroups": {
+      "items": {
+        "type": "string"
+      },
+      "type": "array"
+    },
+    "defined": {
+      "type": "boolean"
     },
     "groups": {
-      "uniqueItems": true,
-      "type": "array",
-      "description": "The groups associated with the application role.",
-      "items": { "type": "string" }
+      "items": {
```

Comment:
format

Reply:
applied but somewhy it does not available in commit diff (available in global diff)